templates/instantanswer/ia.tx

```diff
@@ -42,61 +42,47 @@
-                    </li>
-                    <li>
-                        <h3 class="ia-single--header"> Set up your development environment </h3>
-                        <p> 
-                            <a href="https://duck.co/duckduckhack/setup_dev_environment"> Here </a> you'll find a quick explanation. 
-                        </p>
-                    </li>
-                    <li>
-                        <h3 class="ia-single--header">Create a Pull Request </h3>
-                        <p> Don't forget to include the link to this 
-                            <a href="https://duck.co/ia/view/<: $id :>"> IA Page</a>
-                            in the PR description.<br/>
-                            Dax will post to your PR from time to time: when you open your PR, when it gets deployed
-                            to the beta server, and when it's merged. 
-                        </p>
-                    </li>
-                </ul>
-                <p>
-                    If you have any questions, feel free to ask them on our 
-                        <a href="https://duckduckhack.slack.com/messages/ask-anything/">Slack channel</a>.<br />
-                    Thanks for contributing, we hope you'll have fun!
-                </p>
-                <div class="button btn--primary" id="next-steps-close"> Ok </div>
-                <span class="clearfix"></span>
-            </div>
-        <: } :>
+      <div class="infobox hide" data-contributor="<: if $new_contributor { :>new<: } else { :>old<: } :>">
+	<header class="fix">
+	  <h1 class="pull-left tx-size--medium tx-clr--dk">Next Steps</h1>
+	  <span class="ddgsi-close-bold tx-size--medium pull-right"></span>
+	</header>
+	<div class="gw infobox__container">
+	  <div class="g third">
+	    <h2 class="tx-size--small tx-clr--dk">
+	      <strong>Set Up Your Dev Environment</strong>
+	    </h2>
+	    <p>
+	      <a href="https://duck.co/duckduckhack/setup_dev_environment"> Here </a> we'll give a step-by-step guide on how to get your development environment ready.
+	    </p>
+	    <p>
+	      If you have any questions, feel free to ask them on our 
+              <a href="https://duckduckhack.slack.com/messages/ask-anything/">Slack channel</a>!
+	    </p>
+	  </div>
+	  <div class="g third">
+	    <h2 class="tx-size--small tx-clr--dk">
+	      <strong>Fork the Repo</strong>
+	    </h2>
+	    <p>
+	      After getting your environment set up, make sure you fork the right repo to start coding. Check out <a href="https://duck.co/duckduckhack/setup_dev_environment#fork-the-correct-duckduckhack-repository-on-github"> the docs </a>
+              for more information.
+	    </p>
+	  </div>
+	  <div class="g third">
+	    <h2 class="tx-size--small tx-clr--dk">
+	      <strong>Create a Pull Request</strong>
+	    </h2>
+	    <p>
+	      When everything is ready, send us your code by creating a <a href="https://duck.co/duckduckhack/preparing_for_a_pull_request">pull request</a>.
```

Comment:
change to

When your IA is ready for initial review, [create a pull request](https://duck.co/duckduckhack/preparing_for_a_pull_request). Be sure to paste the link to this IA page in the PR description:

IA: [title of this IA](link to this actual IA page)